$FreeBSD$

- make sure link pulls in libstdc++ (jam link invoked as cc instead of c++)
- don't name shlib files with extra trailing .0

--- Jamrules.orig	2011-02-22 09:10:30.000000000 -0700
+++ Jamrules	2012-02-09 17:14:48.000000000 -0700
@@ -33,6 +33,10 @@
       case CYGWIN :
          LINKLIBS += -lstdc++ ;
 
+      case FREEBSD :
+         LINKLIBS += -lstdc++ ;
+         CCSHARE_FLAGS = -fPIC ;
+
       case LINUX :
          LINKLIBS += -lstdc++ ;
          CCSHARE_FLAGS = -fPIC ;
@@ -133,7 +137,7 @@
 # compiler specific stuff - this is probably actually pretty clumsy
 if ( $(CC) != gcc )
 {
-   NEED_GETOPT = true ;
+   #NEED_GETOPT = true ;
 }
 
 if $(PREFIX)
@@ -238,7 +242,7 @@
    if $(UNIX) && $(OS) != CYGWIN
    {
      soname = $(LIBPREFIX)$(<:S=$(SUFLIBSHR)).$(GPSTK_SO_VERSION) ;
-     shlib = $(LIBPREFIX)$(<:S=$(SUFLIBSHR).$(GPSTK_SO_VERSION).0) ;
+     shlib = $(LIBPREFIX)$(<:S=$(SUFLIBSHR)).$(GPSTK_SO_VERSION) ;
      shrobjs = $(>:S=$(SUFOBJ2)) ;
      ObjectCcFlags $(shrobjs) : $(CCSHARE_FLAGS) ;
      ObjectC++Flags $(shrobjs) : $(CCSHARE_FLAGS) ;
